### DISCUSSION TITLE
Integration tests for adapters

### DISCUSSION THREAD

**Discussion by penx** [remix-run/remix#3401] [Proposals]
It would be good to have a way to run a set of integration tests against each adapter.

While working on the google cloud functions adapter I was able to temporarily switch the fixtures to use my adapter and run the existing integration tests, but this isn't done in a way that could be merged in:

https://github.com/penx/remix/commit/a9d8fa694b913a76adeb45e500f69d3da7e852fe

I'm happy to work on this but open to suggestions on if it's wanted and how to approach it.

**Comment by MichaelDeBoey** [comment]
I think it's a great idea to have tests running against all different adapters.

We should make sure to be able to test tiny differences though.
